sg: db add-user panics

```
~/work/sourcegraph  main $ sg db add-user -h
add-user flag redefined: username
panic: add-user flag redefined: username

goroutine 1 [running]:
flag.(*FlagSet).Var(0x14000624480, {0x1029b0fe8, 0x1400044bd70}, {0x140008e5410, 0x8}, {0x101c45738, 0x11})
	flag/flag.go:879 +0x2d8
flag.(*FlagSet).StringVar(...)
	flag/flag.go:762
flag.(*FlagSet).String(0x14000624480, {0x140008e5410, 0x8}, {0x101c5ee10, 0x16}, {0x101c45738, 0x11})
	flag/flag.go:775 +0xa0
github.com/urfave/cli/v2.(*StringFlag).Apply(0x10350bca0, 0x14000624480)
	github.com/urfave/cli/v2@v2.4.0/flag_string.go:75 +0x1b0
github.com/urfave/cli/v2.flagSet({0x101c032dc, 0x8}, {0x14000153400, 0x3, 0x4})
	github.com/urfave/cli/v2@v2.4.0/flag.go:134 +0x158
github.com/urfave/cli/v2.(*Command).newFlagSet(...)
	github.com/urfave/cli/v2@v2.4.0/command.go:172
github.com/urfave/cli/v2.(*Command).parseFlags(0x10350f520, {0x1029ce0b0, 0x14000142e10}, 0x0)
	github.com/urfave/cli/v2@v2.4.0/command.go:180 +0x4c
github.com/urfave/cli/v2.(*Command).Run(0x10350f520, 0x140001533c0)
	github.com/urfave/cli/v2@v2.4.0/command.go:106 +0x208
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0x14000582b60, 0x14000471ec0)
	github.com/urfave/cli/v2@v2.4.0/app.go:434 +0x8d4
github.com/urfave/cli/v2.(*Command).startApp(0x10350f1c0, 0x14000471ec0)
	github.com/urfave/cli/v2@v2.4.0/command.go:280 +0x7f0
github.com/urfave/cli/v2.(*Command).Run(0x10350f1c0, 0x14000471ec0)
	github.com/urfave/cli/v2@v2.4.0/command.go:94 +0x798
github.com/urfave/cli/v2.(*App).RunContext(0x103511b60, {0x1029c6190, 0x140000400c0}, {0x1400003c080, 0x4, 0x4})
	github.com/urfave/cli/v2@v2.4.0/app.go:313 +0x760
main.main()
	github.com/sourcegraph/sourcegraph/dev/sg/main.go:24 +0xdc
```

--- a/dev/sg/sg_db.go
+++ b/dev/sg/sg_db.go
@@ -64,7 +64,7 @@ var (
 						Usage: "Username for user",
 					},
 					&cli.StringFlag{
-						Name:  "username",
+						Name:  "password",
 						Value: "sourcegraphsourcegraph",
 						Usage: "Password for user",
 					},

